cmd/virt-launcher/qemu

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+if [ "$2" != "release" ] || [ "$3" != "end" ] || [ "$4" != "migrated" ]; then
+	exit
+fi
+touch /run/kubevirt-private/backend-storage-meta/migrated
```

Comment:
Does this ever need to be cleaned up again?

Reply:
In RWO, a migrated backend-storage PVC is obsolete and will be removed entirely anyway.
In RWX, we don't perform recoveries so the file will be ignored forever.
So I think we're good.

Reply:
Do I understand correctly that the `backend-storage-meta` will be present in all the VMs in a second phase? 
The issue this PR is partially solving also affects volume migration. It be possible that the backend storage is on a RWX storage class but still the destination and source volumes are affected by the race since it is always a copy. In that case, we always need to have a recovery.

Reply:
> Do I understand correctly that the `backend-storage-meta` will be present in all the VMs in a second phase?

Yes

> The issue this PR is partially solving also affects volume migration. It be possible that the backend storage is on a RWX storage class but still the destination and source volumes are affected by the race since it is always a copy. In that case, we always need to have a recovery.

If backend-storage is RWX, it is assumed all storage is RWX... If we need to handle the case where backend-storage is RWX and VM storage is RWO, I think it should be a separate effort, as this PR is already really big...